hips/hip-999.md

```diff
@@ -0,0 +1,75 @@
+---
+hip: 9999
+title: "Transitioning Security Team Members on Project Deprecation"
+authors: [ "Matt Butcher <matt.butcher@microsoft.com>" ]
+created: "2020-09-24"
+type: "process"
+status: "draft"
+---
+
+## Abstract
+
+Helm maintains a security team composed of interested maintainers from Helm projects. This
+proposal clarifies that to be a member of the security team, one must be a maintainer of
+an active (non-deprecated) project.
+
+## Motivation
+
+The existing security team documentation does not cover cases where a Helm project is
+marked deprecated. However, because the security team discusses sensitive and potentially
+damaging issues, the Helm project requires clarity on this point.
+
+## Rationale
+
+This HIP suggests that security team members must be maintainers on _active_ Helm projects.
+
+## Specification
+
+According to the current documentation in [SECURITY.md], the only requirements for
```

Comment:
The correct format for these links is apparently [TEXT][REFERENCE] which is not what I was doing in 003 ([until today](https://github.com/helm/community/pull/149/commits/148bcc6878d6c11de99fab2a5d06ab5fcb387720)) but is what happens in 001 and 002.